Automattic/vip-go-mu-plugins
# [CANTINA-889] Add Prometheus plugin

## Description

This PR introduces Prometheus plugin and several basic collectors. Later, I am going to add more collectors.

## Changelog Description

### Plugin Added: Prometheus

Added Prometheus plugin to collect performance metrics.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out the code
2. Create a dev env with mu-plugins pointing to the checked-out code.
3. Go to `http://vip-local.vipdev.lndo.site:8000/metrics` (or whatever the URL is) and refresh the page a couple of times.

![metrics](https://user-images.githubusercontent.com/7810770/180807557-a1d94e81-4e1d-41bc-962f-5bd9b7270afe.png)
